src/bp/ui-studio/src/web/views/OneFlow/sidePanel/TopicList/index.tsx

```diff
@@ -301,45 +287,19 @@ const TopicList: FC<Props> = props => {
           <span>
             {displayName} {type !== 'qna' ? tooltip : qnaTooltip}
           </span>
-          <div className={style.overhidden} id="actions">
-            {type !== 'qna' && (
-              <Fragment>
-                <Tooltip
-                  content={<span>{lang.tr('studio.flow.topicList.editWorkflow')}</span>}
-                  hoverOpenDelay={500}
-                  position={Position.BOTTOM}
-                >
-                  <Button icon="edit" minimal onClick={editWorkflow} />
-                </Tooltip>
-                <Tooltip
-                  content={<span>{lang.tr('studio.flow.topicList.deleteWorkflow')}</span>}
-                  hoverOpenDelay={500}
-                  position={Position.BOTTOM}
-                >
-                  <AnchorButton icon="trash" minimal onClick={deleteWorkflow} disabled={lockedFlows.includes(name)} />
-                </Tooltip>
-              </Fragment>
-            )}
-            {type === 'qna' && (
-              <Tooltip
-                content={<span>{lang.tr('studio.flow.topicList.editQna')}</span>}
-                hoverOpenDelay={500}
-                position={Position.BOTTOM}
-              >
-                <Button icon="edit" minimal onClick={editQnA} />
-              </Tooltip>
-            )}
-          </div>
         </div>
       ),
-      icon
+      icon: 'none'
     }
   }
 
   const onClick = (el: NodeData | string, type) => {
     if ((el as NodeData)?.type === 'qna') {
       // Return true will mimic preventDefault for TreeView's onClick
       return true
+    } else if ((el as NodeData)?.type === 'adder') {
+      props.createWorkflow((el as any).topic)
```

Comment:
Please avoid any of there's a typing available, if your property is not there, you can add it.